ee/dtr/architecture.md

```diff
@@ -11,76 +11,62 @@ Docker Universal Control Plane cluster.
 
 ![](images/architecture-1.svg)
 
-Once you have DTR deployed, you use your Docker CLI client to login, push, and
-pull images.
+Following DTR deployment use the Docker CLI client to login, push, and pull images.
 
-## Under the hood
+## High availability
 
-For high-availability you can deploy multiple DTR replicas, one on each UCP
+To ensure high-availability, deploy multiple DTR replicas, one on each UCP
 worker node.
 
 ![](images/architecture-2.svg)
 
-All DTR replicas run the same set of services and changes to their configuration
-are automatically propagated to other replicas.
+All DTR replicas run the same set of services, and any changes made to the configuration of one replica is automatically propagated to the other replicas.
 
 ## DTR internal components
 
-When you install DTR on a node, the following containers are started:
+Installing DTR on a node starts a selection of containers. Be aware that these containers are for the **internal DTR use ONLY** and are not intended for application use.
 
-| Name                                 | Description                                                                                                                        |
+| Container Name                                 | Description                                                                                                                        |
 |:-------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------|
-| dtr-api-&lt;replica_id&gt;           | Executes the DTR business logic. It serves the DTR web application, and API                                                        |
+| dtr-api-&lt;replica_id&gt;           | Executes DTR business logic;  serves the DTR web application and API                                                        |
 | dtr-garant-&lt;replica_id&gt;        | Manages DTR authentication                                                                                                         |
 | dtr-jobrunner-&lt;replica_id&gt;     | Runs cleanup jobs in the background                                                                                                |
-| dtr-nginx-&lt;replica_id&gt;         | Receives http and https requests and proxies them to other DTR components. By default it listens to ports 80 and 443 of the host   |
-| dtr-notary-server-&lt;replica_id&gt; | Receives, validates, and serves content trust metadata, and is consulted when pushing or pulling to DTR with content trust enabled |
+| dtr-nginx-&lt;replica_id&gt;         | Receives http and https requests and proxies these requests to other DTR components; listens to ports 80 and 443 of the host, by default   |
```

Comment:
dtr-jobrunner should be "Runs cleanup, scanning, mirroring and other background jobs"